src/DotNetEnv/LoadOptions.cs

```diff
@@ -4,47 +4,43 @@ namespace DotNetEnv
 {
     public class LoadOptions
     {
-        public static readonly LoadOptions DEFAULT = new LoadOptions();
-
-        public bool SetEnvVars { get; }
-        public bool ClobberExistingVars { get; }
-        public bool OnlyExactPath { get; }
-
-        public LoadOptions(
-            bool setEnvVars = true,
-            bool clobberExistingVars = true,
-            bool onlyExactPath = true
-        ) {
-            SetEnvVars = setEnvVars;
-            ClobberExistingVars = clobberExistingVars;
-            OnlyExactPath = onlyExactPath;
-        }
+        public bool SetEnvVars { get; set; } = true;
+        public bool ClobberExistingVars { get; set; } = true;
+        public bool OnlyExactPath { get; set; } = true;
 
-        public LoadOptions(
-            LoadOptions old,
-            bool? setEnvVars = null,
-            bool? clobberExistingVars = null,
-            bool? onlyExactPath = null
-        ) {
-            SetEnvVars = setEnvVars ?? old.SetEnvVars;
-            ClobberExistingVars = clobberExistingVars ?? old.ClobberExistingVars;
-            OnlyExactPath = onlyExactPath ?? old.OnlyExactPath;
-        }
+        public static LoadOptions DEFAULT => new LoadOptions();
 
-        public static LoadOptions NoEnvVars (LoadOptions options = null) =>
-            options == null ? DEFAULT.NoEnvVars() : options.NoEnvVars();
+        public static LoadOptions NoEnvVars(LoadOptions options = null) =>
+            (options ?? DEFAULT).NoEnvVars();
 
-        public static LoadOptions NoClobber (LoadOptions options = null) =>
-            options == null ? DEFAULT.NoClobber() : options.NoClobber();
+        public static LoadOptions NoClobber(LoadOptions options = null) =>
+            (options ?? DEFAULT).NoClobber();
 
-        public static LoadOptions TraversePath (LoadOptions options = null) =>
-            options == null ? DEFAULT.TraversePath() : options.TraversePath();
+        public static LoadOptions TraversePath(LoadOptions options = null) =>
+            (options ?? DEFAULT).TraversePath();
+    }
 
-        public LoadOptions NoEnvVars () => new LoadOptions(this, setEnvVars: false);
-        public LoadOptions NoClobber () => new LoadOptions(this, clobberExistingVars: false);
-        public LoadOptions TraversePath () => new LoadOptions(this, onlyExactPath: false);
+    public static class LoadOptionExtensions
+    {
+        public static LoadOptions NoEnvVars(this LoadOptions @this)
```

Comment:
Just call this `options` in all these places

Reply:
done